pages/press.md

```diff
@@ -4,32 +4,36 @@ title: Press
 permalink: press
 ---
 
-Are you looking to get a speaker or interview about Solid? Contact us on press@solidproject.org with a description of what you are looking for. 
+Are you looking to get a speaker or interview about Solid? Contact us on [press@solidproject.org](mailto:press@solidproject.org) with a description of what you are looking for.
 
 ## Upcoming Talks
 
 {%
   include talk.html
-    date="2020-01-10"
-    title="P2P"
-    website="https://p2p.paris/fr/event/festival-0/"
-    speaker="Alice Poggioli"
+    date="2020-06-04"
```

Comment:
Unfortunately, this is not an upcoming talk anymore... Maybe we should look at a different organizations of the sections, whith plain chronological order, to reduce the maintenance overhead ?